addon/services/bsy-alert.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
+
+export default Ember.Service.extend({
+  alert: window.swal,
+});
```

Comment:
I know we talked about this already but I upgraded the ember version in #85 so we can create the sweetalert shim and use the `import` syntax once that's merged.